src/Interpreters/Context.cpp

```diff
@@ -1813,6 +1829,56 @@ zkutil::ZooKeeperPtr Context::getZooKeeper() const
     return shared->zookeeper;
 }
 
+#if USE_HIVE
+HiveMetastoreClientPtr Context::getHiveMetastoreClient(const String & name) const
```

Comment:
A reminder that need to move this out of context.

Reply:
done